src/Avatar/Avatar.js

```diff
@@ -101,7 +113,7 @@ Avatar.propTypes = {
     'size18',
   ]),
   /** Background color when text (initials) content is displayed */
-  color: PropTypes.oneOf(['blue', 'green', 'grey', 'red', 'orange']),
+  color: PropTypes.oneOf(avatarColorList),
```

Comment:
I think our AutoDocs doesn't support this case, because:
![image](https://user-images.githubusercontent.com/7141972/65148069-240eaf80-da28-11e9-868e-c0020bcdf3e5.png)

Try to:
```suggestion
  color: PropTypes.oneOf([...avatarColorList]),
```

And if it still happens, I would add the values explicitly, like:
```suggestion
  color: PropTypes.oneOf(['A1', 'A2', 'A3', 'A4', 'A5', 'A6',]),
```